### PR TITLE
Fix issue when the quiet mode show some message

### LINKED ISSUE #403: Quiet mode is as quiet as one can expect it to be
| Question    | Answer
| ------------| ---------------
| Infection version | 9.0

Quiet mode is as quiet as it should be. Steps to reproduce:

1. Run `vendor/bin/infection --quiet`
2. Observe being printed out: `You are running Infection with xdebug enabled.`

### PR DESCRIPTION
**Pull request by sidz**
This PR fixes #403 issue and fixes the quiet mode

### CODE REVIEW

**Comment by BackEndTea** [comment]
What if we move this message into the InfectionCommand. That way Symfony will handle whether or not it should output, based on `--quiet`

**Comment by sidz** [comment]
we can't as we show this message before _xdebugHandler_ will do its dirty job :)

File: src/Console/Application.php [hunk 1]
```diff
@@ -100,6 +96,17 @@ public function run(InputInterface $input = null, OutputInterface $output = null
         return parent::run($input, $output);
     }
 
+    private function logRunningWithDebugger(InputInterface $input)
+    {
+        if (!$input->hasParameterOption(['--quiet', '-q'], true)) {
```
Comment (sanmai) [inline src/Console/Application.php]: Shouldn't it be like this instead, what do you think?
```php
    public function logRunningWithDebugger(string $debugger)
    {
        // --- $this->io->writeln(sprintf(self::RUNNING_WITH_DEBUGGER_NOTE, $debugger));
        $this->io->writeln(sprintf(self::RUNNING_WITH_DEBUGGER_NOTE, $debugger), OutputInterface::VERBOSITY_QUIET);
    }
```

Comment (sidz) [reply src/Console/Application.php]: @sanmai yeah sure. Thought about it. 

but symfony will do it a bit later here https://github.com/symfony/console/blob/master/Application.php#L845

**Comment by borNfreee** [comment]
Thank you @sidz

**Pull request merged**